tests/cbmc/proofs/s2n_stuffer_write_uint8/Makefile

```diff
@@ -15,22 +15,25 @@
 
 CBMCFLAGS +=
 
-DEPENDENCIES += $(HELPERDIR)/source/cbmc_utils.c
-DEPENDENCIES += $(HELPERDIR)/source/make_common_datastructures.c
-DEPENDENCIES += $(HELPERDIR)/source/proof_allocators.c
-DEPENDENCIES += $(SRCDIR)/stuffer/s2n_stuffer.c
-DEPENDENCIES += $(SRCDIR)/stuffer/s2n_stuffer_network_order.c
-DEPENDENCIES += $(SRCDIR)/utils/s2n_blob.c
-DEPENDENCIES += $(SRCDIR)/utils/s2n_ensure.c
-DEPENDENCIES += $(SRCDIR)/utils/s2n_mem.c
-DEPENDENCIES += $(SRCDIR)/utils/s2n_safety.c
-
-ENTRY = s2n_stuffer_write_uint8_harness
+HARNESS_ENTRY = s2n_stuffer_write_uint8_harness
+HARNESS_FILE = $(HARNESS_ENTRY).c
+
+PROOF_SOURCES += $(HARNESS_FILE)
+PROOF_SOURCES += $(PROOF_SOURCE)/cbmc_utils.c
+PROOF_SOURCES += $(PROOF_SOURCE)/make_common_datastructures.c
+PROOF_SOURCES += $(PROOF_SOURCE)/proof_allocators.c
+
+PROJECT_SOURCES += $(SRCDIR)/stuffer/s2n_stuffer.c
+PROJECT_SOURCES += $(SRCDIR)/stuffer/s2n_stuffer_network_order.c
+PROJECT_SOURCES += $(SRCDIR)/utils/s2n_blob.c
+PROJECT_SOURCES += $(SRCDIR)/utils/s2n_ensure.c
+PROJECT_SOURCES += $(SRCDIR)/utils/s2n_mem.c
+PROJECT_SOURCES += $(SRCDIR)/utils/s2n_safety.c
 
 # We abstract these functions because manual inspection demonstrates they are unreachable.
-REMOVE_FUNCTION_BODY += --remove-function-body s2n_blob_slice
-REMOVE_FUNCTION_BODY += --remove-function-body s2n_mem_cleanup_impl
-REMOVE_FUNCTION_BODY += --remove-function-body s2n_stuffer_wipe_n
+REMOVE_FUNCTION_BODY += s2n_blob_slice
```

Comment:
Do we want to call it this, or `CBMC_REMOVE_FUNCTION_BODY`?

Reply:
`REMOVE_FUNCTION_BODY` is correct. `CBMC_REMOVE_FUNCTION_BODY` is the name of the variable that contains all functions in `REMOVE_FUNCTION_BODY` concatenated together with `--remove-function-body prepended. See [Makefile.common](https://github.com/awslabs/aws-templates-for-cbmc-proofs/blob/master/template-for-repository/proofs/Makefile.common#L286).